### PR TITLE
Add a (super ugly) panic screen

### PR DESCRIPTION
**Pull request by Dinnerbone**
It's totally lacking but at least now you can see why it crashed without figuring out logcat in time

It's scrollable and selectable, for copy pasting to ~~szolo's DMs~~ a github issue

image

### CODE REVIEW

File: app/src/main/AndroidManifest.xml
Comment (torokati44) [inline app/src/main/AndroidManifest.xml:1]: Hmm, what happened here? Newline type change?

Comment (Dinnerbone) [reply app/src/main/AndroidManifest.xml:1]: Yep, it was crlf before

Comment (torokati44) [reply app/src/main/AndroidManifest.xml:1]: Eww. But also, deciphered the actual change, it's just this bit added:
```xml
        <activity
            android:name=".PanicActivity"
            android:launchMode="singleTask"
            android:exported="false">
        </activity>
```

File: app/src/main/java/rs/ruffle/PanicScreen.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+@Preview(name = "Panic - Dark", uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+fun PanicScreenPreview() {
+    RuffleTheme {
+        PanicScreen(
+            message = """Error: panicked at core/src/display_object/movie_clip.rs:477:9:
+assertion `left == right` failed: Called replace_movie on a clip with LoaderInfo set
+  left: Some(LoaderInfoObject(LoaderInfoObject { ptr: 0x31b30a8 }))
+ right: None
+    at n.wbg.__wbg_new_796382978dfd4fb0 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.90db0a0ab193ed0c601b.js:1:83857)
```
Comment (torokati44) [inline app/src/main/java/rs/ruffle/PanicScreen.kt:65]: _A JS file?! In my native application?! :scream:_

Comment (Dinnerbone) [reply app/src/main/java/rs/ruffle/PanicScreen.kt:65]: I copied a random stacktrace from a recent ruffle issue for somewhat realistic data to visualize 😅

Comment (torokati44) [reply app/src/main/java/rs/ruffle/PanicScreen.kt:65]: I know... ;) :D

File: src/java.rs [hunk 2]
```diff
@@ -172,33 +172,31 @@ impl JavaInterface {
...
-                    .expect("getTraceOutput must exist"),
-                get_loc_in_window: env
-                    .get_method_id(class, "getLocInWindow", "()[I")
-                    .expect("getLocInWindow must exist"),
-                get_android_data_storage_dir: env
-                    .get_method_id(class, "getAndroidDataStorageDir", "()Ljava/lang/String;")
-                    .expect("getAndroidDataStorageDir must exist"),
-            })
-            .unwrap_or_else(|_| panic!("Init cannot be called more than once!"))
+        let _ = JAVA_INTERFACE.set(JavaInterface {
```
Comment (torokati44) [inline src/java.rs:175]: Hmm, did this cause any issues? Why silence it?

Comment (Dinnerbone) [reply src/java.rs:175]: Yeah, I call init multiple times now (to set a valid panic handler) and this method panicked if you did that. So now it doesn't. :D

Comment (torokati44) [reply src/java.rs:175]: I'm not seeing right now, where, or how, but whatever, it's fine.

**Review by torokati44 — APPROVED** [review]
Thank you! ^^

**Pull request merged**